tensorrt_llm/_torch/modules/fused_moe/quantization.py

```
@@ -654,60 +691,97 @@ def setup_quant_scales(self, module: torch.nn.Module):
+                w1_weight_scale = weights[
+                    f"{expert_id}.w1.weight_scale"] if f"{expert_id}.w1.weight_scale" in weights else None
+                w3_weight_scale = weights[
+                    f"{expert_id}.w3.weight_scale"] if f"{expert_id}.w3.weight_scale" in weights else None
+                w2_weight_scale = weights[
+                    f"{expert_id}.w2.weight_scale"] if f"{expert_id}.w2.weight_scale" in weights else None
             elif module.weight_loading_mode == MoEWeightLoadingMode.FUSED_GATE_UP_PROJ:
-                w1_weight_scale = weights[f"gate_up_proj_weight_scale"]
-                w3_weight_scale = weights[f"gate_up_proj_weight_scale"]
-                w2_weight_scale = weights[f"down_proj_weight_scale"]
+                w1_weight_scale = weights[
+                    f"gate_up_proj_weight_scale"] if f"gate_up_proj_weight_scale" in weights else None
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)